### ISSUE TITLE
Mercator Distortions 

### ISSUE THREAD

**Issue by wslafoy** [jimjkelly/missile-defense#15]
Circles do not distort with latitude. 
Low priority, not a big deal since it is just meant for basic simulation, but if easy to implement:
http://gis.stackexchange.com/questions/3264/using-geodesic-measurements-for-circle-radii

**Comment by jimjkelly** [comment]
Ah!  Interesting.  I suppose that makes sense.  I'll have to play around with some of the suggestions there and see what we can come up with.

**Comment by wslafoy** [comment]
Small addition: when the camera passes over the map seam in the pacific ocean, the circles disappear. This is a super common problem with mapping stuff, my bad!

**Comment by jimjkelly** [comment]
Ah!  Interesting. That's kind of a neat bug.  Actually it seems some general badness occurs then, so I'll have to figure out what's going on and fix it.

**Comment by wslafoy** [comment]
Yeah, there are a lot of really interesting mathematically oddities that happen when data crosses the map seam (almost always in the pacific) and gets to polar regions. On google earth, if you drag a polygon towards the poles, it will slowly deform and spiral, but the system will still see it as linear! Its such an interesting artifact  

Fortunately because it is a known and common issue, there will probably be a lot of documentation for fixes
